README.md

```diff
@@ -35,6 +35,28 @@ malcontent has 3 modes of operation:
 
 malcontent is at its best analyzing programs that run on Linux. Still, it also performs admirably for programs designed for other UNIX platforms such as macOS and, to a lesser extent, Windows.
 
+## ⚠️ Malware Disclaimer ⚠️
+
+Due to how malcontent operates, other malware scanners can detect malcontent as malicious. As a general rule of thumb, programs that leverage Yara rules will match other programs that use the same rules due to their strings looking for problematic behaviors.
+
+While not exhaustive, here's an example list of how other scanners see malcontent (based on [this](https://www.virustotal.com/gui/file/b6f90aa5b9e7f3a5729a82f3ea35f96439691e150e0558c577a8541d3a187ba4/detection) VirusTotal scan:
```

Comment:
I think this exhaustive list is going to go stale very very quickly. Can we just remove it and point people to the example issue?

Reply:
Removed in [`5eedaaa` (#728)](https://github.com/chainguard-dev/malcontent/pull/728/commits/5eedaaa418e7f6d5aa0e162e838e6a9722537ee3).